travis/test_server.py

```diff
@@ -204,7 +233,9 @@ def setup_server(db, odoo_unittest, tested_addons, server_path, script_name,
         server_options = []
     print("\nCreating instance:")
     try:
-        subprocess.check_call(["createdb", db])
```

Comment:
Can it be, that you have misunderstood #414? I'm a bit puzzled about your idea...